### DISCUSSION TITLE
i cant install

### DISCUSSION THREAD

**Discussion by zoukba0014** [projectdiscovery/dnsx#243] [Q&A]
github.com/projectdiscovery/dnsx/libs/dnsx: github.com/projectdiscovery/dnsx@v1.1.0: verifying module: checksum mismatch
	downloaded: h1:nHyAI4+1HXevex0ilADtOmhvx7BxctCVmbj91YaV3Vo=
	sum.golang.org: h1:Wjtx4BJQBemIk+Jaibf6bmisPHgovXfX+V1iH+pJ2YU=

**Comment by ehsandeep** [comment]
@zoukba0014 the latest version of dnsx is `v1.1.1`, let me know if you are using the latest version, if not please share steps to reproduce the error.

**Reply by zoukba0014** [reply]
thanks alot i fixed it
